README.md

```diff
@@ -65,20 +65,20 @@ $ ./tools/toolchain/dbuild ./build/release/scylla --help
 
 ## Testing
 
-See [test.py manual](docs/guides/testing.md).
+See [test.py manual](docs/dev/testing.md).
 
 ## Scylla APIs and compatibility
 By default, Scylla is compatible with Apache Cassandra and its APIs - CQL and
 Thrift. There is also support for the API of Amazon DynamoDB™,
 which needs to be enabled and configured in order to be used. For more
 information on how to enable the DynamoDB™ API in Scylla,
 and the current compatibility of this feature as well as Scylla-specific extensions, see
-[Alternator](docs/alternator/alternator.md) and
-[Getting started with Alternator](docs/alternator/getting-started.md).
+[Alternator](docs/source/alternator/alternator.md) and
```

Comment:
What is this "source" subdirectory? I thought you said Alternator will be docs/alternator?

Reply:
@nyh The structure is the same as in other projects that include the docs, for example:
scylla-cloud-doc/docs/source/
java-driver/docs/source/
BUT we've just discussed it and David is going to push another commit to remove the "source" folder from the  structure.

Reply:
@nyh @annastuchlik Commit pushed to remove the source folder.